source/server.ts

```diff
@@ -96,7 +99,9 @@ const getYML = getCachedFormatter(
 
 i18n._formatgetters = { getJS, getJSON, getYML };
 
-const _publishConnectionId = new Meteor.EnvironmentVariable<string | undefined>();
+const _publishConnectionId = new Meteor.EnvironmentVariable<
+  string | undefined
+>();
```

Comment:
Formatting looks a little bit off here, is this intended?

Reply:
I ran ESLint with our latest config here.